libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,22 +6,23 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
+      autocomplete="off"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix aria-label="search">
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
+    <div class="book" data-testing="book-item" *ngFor="let b of books | async">
       <div class="book--title">
         {{ b.title }}
       </div>
       <div class="book--content">
-        <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+        <div class="book--content--cover" role="img" aria-label="cover page">
+          <img src="{{ b.coverUrl }}" alt="Book cover URL" />
```

Comment:
aria-label attribute is used to define a string that labels the current element.

Reply:
You are providing alt attribute to define the book not image url correct?
So you can provide it a meaningful name like `alt="<Book name> book image" or alt="Book image"`